install/README.md

```diff
@@ -8,13 +8,20 @@
 
 ## Prerequisites
 
-Kata Containers requires nested virtualization or bare metal. 
+Kata Containers requires nested virtualization or bare metal.
 See the
 [hardware requirements](https://github.com/kata-containers/runtime/blob/master/README.md#hardware-requirements)
 to see if your system is capable of running Kata Containers.
 
 ## Installing Kata Containers
 
+> **Notes:**
+> - The following guides apply to the latest Kata Containers releases.
+>
+> - Previous stable releases can be installed by replacing the `master`
+> occurrences in the repository URL with a `stable-<version>` from
+> [here](http://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/).
```

Comment:
Could you make it clear in the text of this note that users will need to replace `x86_64` with their architecture if not using that platform?